### PR TITLE
Update other docs name

### PR DESCRIPTION
**Pull request by johypark**
## Summary
Update other docs name in testing-library-with-visual-studio series.

### CODE REVIEW

File: docs/core/tutorials/testing-library-with-visual-studio-code.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ This tutorial shows how to automate unit testing by adding a test project to a s
...
+- This tutorial works with the solution that you create in [Create a .NET Standard library using Visual Studio Code](library-with-visual-studio-code.md).
 
 ## Create a unit test project
 
 Unit tests provide automated software testing during your development and publishing. The testing framework that you use in this tutorial is MSTest. [MSTest](https://github.com/Microsoft/testfx-docs) is one of three test frameworks you can choose from. The others are [xUnit](https://xunit.net/) and [nUnit](https://nunit.org/).
 
 1. Start Visual Studio Code.
 
-1. Open the `ClassLibraryProjects` solution you created in [Create a .NET Standard library in Visual Studio](library-with-visual-studio.md).
+1. Open the `ClassLibraryProjects` solution you created in [Create a .NET Standard library using Visual Studio Code](library-with-visual-studio.md).
```
Comment (tdykstra) [inline docs/core/tutorials/testing-library-with-visual-studio-code.md:20]:
```suggestion
1. Open the `ClassLibraryProjects` solution you created in [Create a .NET Standard library using Visual Studio Code](library-with-visual-studio-code.md).
```

**Review by tdykstra — APPROVED** [review]
Thanks!   I made a suggestion to fix one preexisting error.

**Comment by JohyPark** [comment]
@tdykstra Thanks for review comment.
Checks failed because of #20113.

**Comment by Youssef1313** [comment]
> 
> 
> @tdykstra Thanks for review comment.
> Checks failed because of #20113.

These are being fixed in #20396

**Pull request merged**